ReactWindows/ReactNative/ReactNativeViewHost.cs

```diff
@@ -0,0 +1,79 @@
+using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using ReactNative.Common;
+using ReactNative.Modules.Core;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ReactNative
+{
+    /// <summary>
+    /// Simple class that holds the <see cref="ReactNativeAppHost"/> used in the multi window applications. 
+    /// </summary>
+    public abstract class ReactNativeViewHost : IAsyncDisposable
```

Comment:
>ReactNativeViewHost [](start = 26, length = 19)

As I said in `ReactNativeHost`, I'm not convinced we need a class hierarchy to achieve all this.